pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -556,8 +556,6 @@ func (m *kubeGenericRuntimeManager) killContainer(pod *v1.Pod, containerID kubec
 		gracePeriod = *pod.Spec.TerminationGracePeriodSeconds
 	}
 
-	klog.V(2).Infof("Killing container %q with %d second grace period", containerID.String(), gracePeriod)
```

Comment:
nit: We need a log line to know that prestop hooks are being executed. If it's already there, ignore my comment.

Reply:
it is already there
https://github.com/kubernetes/kubernetes/blob/e2be7c91d956811c98433dd6d72c4ea1a4342905/pkg/kubelet/kuberuntime/kuberuntime_container.go#L464-L465